filebeat/harvester/source/file.go

```diff
@@ -7,3 +7,4 @@ type File struct {
 }
 
 func (File) Continuable() bool { return true }
+func (File) HasState() bool    { return true }
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method File.HasState should have comment or be unexported